rca/project_styleguide/templates/patterns/pages/landingpage/landing_page--research.html

```diff
@@ -103,13 +103,16 @@ <h2 class="section__heading section__heading--primary heading heading--four">
                     <div class="section__row section__row--first section__row--last">
                         {% include "patterns/organisms/image-video-block/image-video-block.html" with modal=false curriculum=false subheading=false has_meta=True heading=featured_image.title meta_heading=featured_image.subtitle meta_copy=featured_image.description href=featured_image.get_link_url image=featured_image.image modifier="tight-heading" %}
                     </div>
+
+                    {% if slideshow_block or page.cta_block or page.news %}
                     <div class="section__notch">
                         <div class="section__notch-fill section__notch-fill--second-col"></div>
                     </div>
+                    {% endif %}
                 </section>
             {% endif %}
 
-            {% if slideshow_block or page.text_teaser %}
```

Comment:
This is the fix for the first issue where text promo (cta_block) does not appear. No idea where `text_teaser` is since I can't find it in this template.

Reply:
Thanks for the context!